crates/bevy_asset/src/server/mod.rs

```diff
@@ -867,6 +873,11 @@ pub enum AssetLoadError {
         loader: &'static str,
         error: AssetLoaderError,
     },
+    #[error("The asset at '{base_path}' does not contain the labeled asset '{label}'.")]
```

Comment:
I think this phrasing could be a tad confusing. Maybe "the file at x does not contain the asset"?

Reply:
I was trying to avoid the word "file" as assets are just bytes that can come from anywhere. But now that I've thought about it a bit, I think "file" here is still the least confusing thing.